task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -8,24 +8,32 @@ static void Main(string[] args)
         {
             Console.WriteLine("Enter A Number :");
             int inputNumber = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter A Number :");
```

Comment:
Maybe would it be better to write smth like "Enter the first number" and "Enter the second number"?